astro/src/content/docs/get-started/download-and-install/_docker-compose-files.mdx

```diff
@@ -20,5 +20,3 @@ docker compose up -d
 docker compose logs -f
 ```
 
-Now you can visit `http://localhost:9011` and configure FusionAuth.
```

Comment:
Why did this get removed? Seems like we still need it, because that is telling someone to log in to the admin UI and configure FusionAuth. 

What am I missing?

Reply:
I found it confusing that there was a link to the localhost both before and after the "done" message:

<img width="647" alt="image" src="https://github.com/user-attachments/assets/678c1a19-073a-463e-b026-28b49b84286d">